priv/worker.py

```diff
@@ -33,7 +49,7 @@
     year = {2016},
     publisher = {GitHub},
     journal = {GitHub repository},
-    howpublished = {\url{https://github.com/openai/gym-http-api}}
+    howpublished = {\\url{https://github.com/openai/gym-http-api}}
```

Comment:
I checked out if there is a solution but found none other than using double dashes or removing the \u from the source file. So should I change it to following?
```
howpublished = {https://github.com/openai/gym-http-api}
```